src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -541,30 +548,30 @@ private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken r
                 switch (modifier.Kind())
                 {
                     case SyntaxKind.OutKeyword:
-                        refOrOutKeyword = modifier;
+                        refnessKeyword = modifier;
                         if (refKind == RefKind.None)
                         {
                             refKind = RefKind.Out;
                         }
                         break;
                     case SyntaxKind.RefKeyword:
-                        refOrOutKeyword = modifier;
+                        refnessKeyword = modifier;
                         if (refKind == RefKind.None)
                         {
                             refKind = RefKind.Ref;
                         }
                         break;
                     case SyntaxKind.InKeyword:
+                        refnessKeyword = modifier;
```

Comment:
>refnessKeyword = modifier; [](start = 24, length = 26)

This code feels wrong. Why refnessKeyword is changed unconditionally, but refKind is changed conditionally?